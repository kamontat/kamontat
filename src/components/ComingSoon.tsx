import React from "react"

// TODO: Implement coming soon component
const ComingSoon = (): JSX.Element => <div>Hello world</div>

export default ComingSoon
