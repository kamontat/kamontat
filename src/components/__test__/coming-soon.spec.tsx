import React from "react"
import renderer from "react-test-renderer"
import ComingSoon from "../ComingSoon"

describe("ComingSoon", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<ComingSoon />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
