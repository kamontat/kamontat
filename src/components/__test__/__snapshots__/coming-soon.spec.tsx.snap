// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ComingSoon renders correctly 1`] = `
<div>
  Hello world
</div>
`;
